### DISCUSSION TITLE
Obliteration compatibility list.

### DISCUSSION THREAD

**Discussion by SoffiMeower** [obhq/obliteration#712] [General]
https://docs.google.com/spreadsheets/d/1FSr7vXfhMcIb5JfZ2R6O6sY4Ki7yKjLFOyogvwHguwU/edit

Made from my Kyty compatibility template, I now set up a sheet for Obliteration!
Anyone can contribute! (The examples below for the features are from the Kyty spreadsheet.)

- Statuses are autocapitalized and color-coded. 
- Spreadsheet is sorted by alphabetical name. 
![image](https://user-images.githubusercontent.com/45863583/193019231-9c8c0268-8d01-46e2-9b61-8eceef539613.png)
- PS4 and PS5 are color coded (PS4 is light blue, PS5 dark blue.)
![image](https://user-images.githubusercontent.com/45863583/193019154-f2fee3bd-a6b5-4fd4-85df-d8e036fef383.png)
- Game tests with the latest emulator version have green text, while older emulator versions have red text.
![image](https://user-images.githubusercontent.com/45863583/193019464-5431f0fb-a204-468e-9a63-a5be19e4283a.png)
- Compatibility counter updates in real time! (Uses an COUNTIF statement on the status region)
![image](https://user-images.githubusercontent.com/45863583/193019909-5e0890b9-c1b5-4326-bbe6-ccd510f7c1ee.png)
![image](https://user-images.githubusercontent.com/45863583/193019795-9b1d9319-db8f-4805-a21d-c71d9181d9c3.png)
- Changes color depending on if a game was tested less than a week ago, a week or longer, a month or longer, and a year or longer.
![image](https://user-images.githubusercontent.com/45863583/193020392-a3bd42cf-edbd-4494-8f1c-84722abf5852.png)

**Comment by SoffiMeower** [comment]
Making this as an issue as idk where you wanna put the link :P

**Comment by ultimaweapon** [comment]
I think we should create a static website instead of Google Spreadsheet and let's people contribute by create a PR. With this way it is easier to use and we can prevent bad people from destroying it.

**Comment by SoffiMeower** [comment]
> I think we should create a static website instead of Google Spreadsheet and let's people contribute by create a PR. With this way it is easier to use and we can prevent bad people from destroying it.

Most likely a Jekyll page using Github Pages?

**Comment by ultimaweapon** [comment]
Something like that. It will even better we can use a full SPA framework like React.js or Vue.js but I'm not sure if GitHub Page support it.

**Comment by SoffiMeower** [comment]
> Something like that. It will even better we can use a full SPA framework like React.js or Vue.js but I'm not sure if GitHub Page support it.

It does, as long as its a static page.

**Comment by ultimaweapon** [comment]
The main problem is both React and Vue required build tool to make development easier. For Jekyll GitHub will run for us but what about React or Vue?

**Comment by SoffiMeower** [comment]
https://github.com/gitname/react-gh-pages

**Comment by ultimaweapon** [comment]
That one use a manual step to deploy to GitHub Page. What I mean is we push page source to the repository and GitHub will build for us automatically like Jekyll.

**Comment by SoffiMeower** [comment]
https://dev.to/achukka/deploy-react-app-using-github-actions-157d
This requires admin perms for keys, so have fun with this .w.

**Comment by ultimaweapon** [comment]
This one looks good except one thing, which is allow Actions to access SSH key that capable pushing to every branches on the repository. Let's say someone create a PR and successfully extract this SSH key now they can destroy the whole repository.

**Comment by SevastianGit** [comment]
Huh, PS5 tag? Does this mean this emulator, like Kyty, should be able to boot both PS4 and PS5 stuff?

**Comment by ultimaweapon** [comment]
No any plan for PS5 yet.

**Comment by SoffiMeower** [comment]
> Huh, PS5 tag? Does this mean this emulator, like Kyty, should be able to boot both PS4 and PS5 stuff?

Not in a long while. We don't even have PS4 done yet XD

**Comment by kairusds** [comment]
You can just use actions for deploying vuejs to pages, the tokens used by actions are temporary and doesn't show up in the logs https://github.com/marketplace/actions/vue-to-github-pages

**Comment by ultimaweapon** [comment]
Thanks for the information.

**Comment by SoffiMeower** [comment]
Updated the sheet haha

**Comment by ultimaweapon** [comment]
I got (maybe) a bad news. I need to find a full-time job within this month. What the meaning of this is once I started to work on my full-time job it is likely that I'll have some spare time to work on Obliteration only weekends.

**Comment by SoffiMeower** [comment]
> I got (maybe) a bad news. I need to find a full-time job within this month. What the meaning of this is once I started to work on my full-time job it is likely that I'll have some spare time to work on Obliteration only weekends.

Well, you know what you must do, right? **S p e e d r u n**
![d8d](https://github.com/obhq/obliteration/assets/45863583/552e9e22-59cc-4484-8ca9-4ddad24a0381)

**Comment by ultimaweapon** [comment]
Superseded by #418.
